### PR TITLE
Docker container for the web app

### PR DESCRIPTION
**Pull request by Pasarus**
Adds all the necessary bits that create a docker container containing the head.mha.

Bind the ports as necessary, the internal port that needs to be bound is 8080 so for example:
`docker run -p 8080:8080 <container-id>`

### CODE REVIEW

File: docker/web-app/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
+ARG BASE_IMAGE=ubuntu:20.04
+FROM ${BASE_IMAGE} as base
+
+LABEL \
+  author.name="Edoardo Pasca, Laura Murgatroyd , Samuel Jones" \
+  author.email=edoardo.pasca@stfc.ac.uk \
+  maintainer.email=edoardo.pasca@stfc.ac.uk \
+  maintainer.url=https://www.ccpsynerbi.ac.uk/  \
```
[the remark this answers is not in the document]
Comment (Pasarus) [reply docker/web-app/Dockerfile:8]: Good catch!

**Comment by Pasarus** [comment]
- I'd like to leave it as right now the docker script is dependent on it
- I have removed it, I am not sure what it is for.

**Comment by paskino** [comment]
I think, as @lauramurgatroyd pointed out, that the head should not be part of this repository.

It can be downloaded to the docker image together with its [license ](https://github.com/TomographicImaging/CIL-Data/blob/main/VTKData%20LICENSE.txt)from [CIL-Data](https://github.com/TomographicImaging/CIL-Data/blob/main/headsq.mha). 

BTW I just noticed that the head is called `headsq.mha` in CIL-Data. Maybe we should rename it to `head.mha` and force push, release?

**Comment by lauramurgatroyd** [comment]
please see https://github.com/TomographicImaging/CIL-Data/pull/11

**Review by lauramurgatroyd — APPROVED** [review]
Working on windows and nice README thanks 💯

**Pull request merged**